apps/privacy-panel/templates/tc/applications.html

```diff
@@ -8,7 +8,15 @@ <h1 data-l10n-id="tc-applications">Applications</h1>
     </header>
 
     <div>
-      <ul>
+      <form id="tc-appSearch" role="search">
+        <button type="submit" data-l10n-id="tc-search-close">close</button>
+        <p>
+          <input type="text" data-l10n-id="tc-search-apps" placeholder="Search Apps" dir="auto">
+          <button type="reset" data-l10n-id="tc-search-clear">Clear</button>
```

Comment:
the clear button isn't working for me (removing the focus (and the keyboard) but not actually clearing the field)